paltas/Sampling/distributions.py

```diff
@@ -338,7 +338,7 @@ def __init__(self, z_lens_min,z_lens_mean,z_lens_std,z_source_min,
 		z_source_mean,z_source_std):
 		# transform z_lens_min, z_source_min to be in units of std. deviations
 		self.z_lens_min = (z_lens_min - z_lens_mean) / z_lens_std 
-		self.z_source_min = (z_source_min - z_source_mean) / z_source_std
+		self.z_source_min_default = (z_source_min - z_source_mean) / z_source_std
```

Comment:
I would keep the original naming convention and instead change the object used in the __call__